.github/scripts/amd/patch_triton_wheel.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -x
+
+WHEELHOUSE_DIR=/artifacts
+PATCHELF_BIN=patchelf
+ROCM_LIB=backends/amd/lib
+ROCM_LD=backends/amd/llvm/bin
+PREFIX=triton
+fname_without_so_number() {
+    LINKNAME=$(echo $1 | sed -e 's/\.so.*/.so/g')
+    echo "$LINKNAME"
+}
+
+replace_needed_sofiles() {
+    find $1 -name '*.so*' -o -name 'ld.lld' | while read sofile; do
+        origname=$2
+        patchedname=$3
+        if [[ "$origname" != "$patchedname" ]] || [[ "$DESIRED_CUDA" == *"rocm"* ]]; then
```

Comment:
Strictly speaking, no need to check for `|| [[ "$DESIRED_CUDA" == *"rocm"* ]`
```suggestion
        if [[ "$origname" != "$patchedname" ]]; then
```